### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_WAT: Skip link is not provided for the application to skip repetative blocks. Issue is observed through out the application

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#115]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites;  #WCAG2.4.1; #C+E-H; #Edge

**Environment Details:**
**URL:** https://manage.iis.net/webserver/central-certificate-store
**Browser Details:** Edge(42.17134.1.0)
Web Accessibility Toolbar

**Repro Steps:**
1.Download and install "Microsoft IIS Administration" application.
2.Enable Internet information Services.
3.Open "Microsoft IIS" application using the URL: https://manage.iis.net/.
4.Enable WAT.
5.DocInfo-->Skip Links.
6.Verify any skip link is provided for application.

**Actual Result:**
Skip link is not provided for the application to skip repetitive blocks.

**Expected Result:**
Skip link should be provided application to skip repetitive blocks.

**User Impact:** 
Keyboard only users need to go through every tab even if it is not necessary for them if skip link is not provided.

**Attachment:**
![bug_115_skip link is not provided](https://user-images.githubusercontent.com/38523250/44704392-bd36f500-aab8-11e8-8dcd-373babb1829f.png)

**Comment by jhkimnew** [comment]
Won'fix.
FYI,
In WAC mode, users can move to the content page with a single tab key.

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
